src/gui/search/searchsortmodel.cpp

```diff
@@ -47,8 +119,47 @@ bool SearchSortModel::lessThan(const QModelIndex &left, const QModelIndex &right
         if (Utils::String::naturalSort(vL.toString(), vR.toString(), res))
             return res;
 
-        return QSortFilterProxyModel::lessThan(left, right);
+        return base::lessThan(left, right);
+    }
+
+    return base::lessThan(left, right);
+}
+
+bool SearchSortModel::filterAcceptsRow(int source_row, const QModelIndex &source_parent) const
+{
+    if (m_isNameFilterEnabled && !m_searchTerm.isEmpty()) {
+        QString name = sourceModel()->data(sourceModel()->index(source_row, NAME)).toString();
+        for (const QString& word: m_searchTermWords) {
+            int i = name.indexOf(word, 0, Qt::CaseInsensitive);
+            if (i == -1) {
+                return false;
+            }
+        }
+    }
+
+    if (m_minSize > 0 || m_maxSize >= 0) {
+        qlonglong size = sourceModel()->data(sourceModel()->index(source_row, SIZE)).toLongLong();
+        if ((m_minSize > 0 && size < m_minSize)
+        ||  (m_maxSize > 0 && size > m_maxSize)) {
```

Comment:
It does not require such alignment (we can't strive to do so in the General case; in this particular case you simply have the variable names of the same length).
You just need to keep (our) rules for Boolean expressions splitting (they have to split before operators) and rules of indentation (4 spaces as indent).